src/main.cpp

```diff
@@ -39,17 +42,37 @@ int main(int argc, char *argv[]){
     int length = 8;
 
     // set symbol set
-    bool symbol_set = false;
+    bool symbol_set = false,save_pass=false;
+
+    //set file path for writing password to current directory as default.
+    std::string file_path ;
 
     for(int i = 1; i<argc; i++) {
-        if(strcmp(argv[i], "-S") == 0) {
+        if(strcmp(argv[i],"-F")==0){
+            if(!save_pass){
+                save_pass = true;
+                file_path =argv[i+1];
+                if(file_path[0]!='/'){
```

Comment:
Why this check? Why are you enforcing absolute paths?

Reply:
Without this check, if the path argument entered is say abcd, then the password was getting saved inside the build directory in a file named abcd.txt

Reply:
Then so be it. There is a more standard way to deal with this. For example, If you run the command `touch abc.txt` it creates the file `abc.txt` in the current directory of the shell. 

Similarly, the case in which only filename is provided, save the file in the current working directory.